### PR TITLE
readme first draft

### PR DESCRIPTION
**Pull request by k-stopczynska**
Switched frontend mentor readme file and created development deploy.
Any suggestions are more than welcomed:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,182 +1,88 @@
-# Frontend Mentor - Invoice app
 
-![Design preview for the Invoice app coding challenge](./preview.jpg)
+# Invoice App
 
-## Welcome! 👋
+Invoicing application that enables users to create, read, updte and delete invoices.
```
Comment (pizgo) [inline README.md]: typo in "update" :)

Comment (k-stopczynska) [reply README.md]: ooops:D

File: README.md [hunk 2]
```diff
@@ -1,182 +1,88 @@
...
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaVsybM7LCJ21R889bhNTgZZrAABk04hHzik4oGw2T6SRfEbWrFhMudckkmHJy5ZFpRjoBacBEd1T1AwOuE8cksoEAM2RO71_QaqsVB9a_vooSddK8A=w2400)
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Create invoice
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaUpup2oQJ7Y2DTHvlT6dceITMrfWIgpEO8P0ePmelandTubT6ipP_G-SSVg75XAMyn7XOjQblzhgpHfxw3Xx7wHJng8SIvmFMAIv0eToWPawib27aU=w2400)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
 
-## Learn More
+## Tech Stack
```
Comment (pizgo) [inline README.md]: I think we could move the Tech Stack section up, eg. before screenshots?

**Comment by pizgo** [comment]
I added some comments in commit + I think we should write that the project's design and idea comes from FrontendMentor :)

File: README.md (hunk 2 above)
Comment (k-stopczynska) [reply README.md]: no problemo, it's also in about section when you enter the repo

**Comment by pizgo** [comment]
Ok, merging approved! ;)

**Pull request merged**